### ISSUE TITLE
Improve Lobby API: room instances

### ISSUE THREAD

**Issue by julienroulle** [nicolodavis/boardgame.io#541]
I think it would be useful to add some infos to the `GET /games/{name}` and `GET /games/{name}/{id}` routes in the Lobby API.

As a user, it would be great to access the setupData passed to the `POST /games/{name}/create`.

The setupData and lobbyConfig informations could be added to the metadata of the game maybe? And then returned through the GET request?

Example:
```
GET /games/{name}/{id}

roomID: the ID of the room instance.

players: the list of seats and players that have joined the game, if any.

setupData: the dictionnary used to initialize the game. 
          (As provided to the /games/{name}/create route)

```

I don't see any problems in that, please tell me if you see any. Otherwise, I would gladly help on implementing this.

**Comment by delucis** [comment]
@julienroulle I can imagine this being helpful. If you're interested in implementing it, check out the `src/server/api.ts` file for the API endpoints.

One thing that [came up recently](https://github.com/nicolodavis/boardgame.io/issues/538#issuecomment-577929848) was that it might be nice to allow players each to provide some custom setup data, which would mean initialising the game with `setupData` after all players had joined. In that case, the setup data returned as you suggest might only be partial.

**Comment by julienroulle** [comment]
resolved in #542
